### ISSUE TITLE
html2canvas can not get background-image which set by css houdini paint api

### ISSUE THREAD

**Issue by smk0621** [niklasvh/html2canvas#3107]
using css houdini paint api to set background-image，as following，but the screenshot is blank content, can not get the background-image 
-----------------------------
@supports ((-webkit-mask: paint(id, type)) or (mask: paint(id, type)))
div {
    background-image: paint(paint-worklet);
}

Does html2canvas support this ？

**Comment by monouowox** [comment]
Yes,so i am.Background-imgae attribute used in css by url() function,function argument is from remote server,use h2htmlcanvas  constructer  pass it options contain 'useCores:true',but i  console it get some logs :Access to image at 'https://***' from origin 'http://192.1***' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.Why use 'useCores:true' cannot resolve cors?
